flake.nix

```diff
@@ -9,7 +9,7 @@
     flake-parts.inputs.nixpkgs-lib.follows = "nixpkgs";
     systems.url = "github:nix-systems/default";
 
-    rust-flake.url = "github:juspay/rust-flake";
+    rust-flake.url = "github:juspay/rust-flake/granular-autoWire";
```

Comment:
@srid We have to update this to use `main` branch again. I had done this change to see if everything is working correctly.

Reply:
Understood. It's okay to use this branch, we can update it to use main next time advancing rust-flake.

That's why I didn't delete the branch of the other PR.